setup.py

```diff
@@ -22,12 +22,12 @@
         "Operating System :: OS Independent",
     ],
     license="Apache 2.0",
-    python_requires='>=3.6',
+    python_requires='>=3.9',
```

Comment:
shouldn't we increase `clarifai-grpc` version from `7.2.0` to `7.3.0` or `8.0.0` ?